.gitignore

```
@@ -2,6 +2,9 @@
 application_default_credentials.json
 .vscode
 .env
+.env.backend.example
+.env.db.example
+.env.proxy.example
```

The example files are correct to be commited, we just don't want the user to commit their real variables to the repository

```suggestion
.env.backend
.env.db
.env.proxy
```